tests/test_aggregate.py

```diff
@@ -8,7 +8,7 @@ def test_aggregate_constructor():
     needed by the constructor?
     """
     config = {}
-    config["FLAVOR"] = None
+    config["FLAVOR"] = "None"
```

Comment:
I guess `FLAVOR` is actually a key name from metadata so in reality it cannot be `None` so the change to string make sense.